tools/gulptasks/jsdoc-dts.js

```diff
@@ -86,13 +87,19 @@ function jsDocESMDTS() {
  */
 function jsDocDTS() {
 
+    const argv = require('yargs').argv;
     const gulpLib = require('./lib/gulp');
     const highchartsDeclarationsGenerator = require(
         '@highcharts/highcharts-declarations-generator'
     );
 
     return new Promise((resolve, reject) => {
 
+        if (argv.custom) {
+            highchartsDeclarationsGenerator.config
+                .mainModule = 'code/custom';
+        }
```

Comment:
I'm getting a `TypeError: Cannot set properties of undefined (setting 'mainModule
')` (even after removing node_modules and reinstalling)